<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройка области графиков</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="cbShowLegend">
     <property name="text">
      <string>показывать легенду</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbMatName">
     <property name="text">
      <string>показывать имя материала в легенде</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbShowGrid">
     <property name="text">
      <string>показывать сетку</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbShowErrorBars">
     <property name="text">
      <string>показывать доверительные интервалы</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbFixedAxisLabels">
     <property name="text">
      <string>зафиксировать подписи осей</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lXAxis">
       <property name="text">
        <string>Ocь X</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leXLabel"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="cbLogScale">
     <property name="text">
      <string>Логарифмическая шкала</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lYAxis">
       <property name="text">
        <string>Ось Y</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leYLabel"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Вид кривых</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="cbLine">
        <property name="text">
         <string>Линия Толщина</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="cbMarker">
        <property name="text">
         <string>Маркер Размер</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="leLineWidth">
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.200000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="leMarkerSize">
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.200000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Отображаемые данные</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Ось X</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cbXaxis"/>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="cbXautoscale">
        <property name="text">
         <string>Автомасштаб</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="leXmin"/>
        </item>
        <item>
         <widget class="QLineEdit" name="leXmax"/>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Ось Y</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="cbYaxis"/>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="cbYautoscale">
        <property name="text">
         <string>Автомасштаб</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLineEdit" name="leYmin"/>
        </item>
        <item>
         <widget class="QLineEdit" name="leYmax"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>451</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>451</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
